include/geometric_shapes/bodies.h

```diff
@@ -446,7 +453,7 @@ class ConvexMesh : public Body
 
   struct MeshData
   {
-    EigenSTL::vector_Vector4f   planes_;
+    EigenSTL::vector_Vector4d   planes_;
```

Comment:
This is going to break ABI, isn't it?
We don't have sonames in this package, do we?